components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -3045,6 +2872,425 @@ subroutine check_values(Qv,T,i,timestepcount,force_abort,source_ind)
     endif
 
   end subroutine check_values
-  !===========================================================================================
+
+  subroutine ice_cld_liquid_collection(rho,t,rhofaci,    &
+  f1pr04,qitot_incld,qc_incld,nitot_incld,nc_incld,    &
+             qccol,nccol,qcshd,ncshdc)
+   
+   !.......................
+   ! collection of droplets
+
+   ! here we multiply rates by air density, air density fallspeed correction
+   ! factor, and collection efficiency since these parameters are not
+   ! included in lookup table calculations
+   ! for T < 273.15, assume collected cloud water is instantly frozen
+   ! note 'f1pr' values are normalized, so we need to multiply by N
+
+
+   implicit none 
+
+   real(rtype), intent(in) :: rho
```

Comment:
I'd like all variable declarations to include a comment giving their units and meaning. Not sure if that should be done in this PR, or in a subsequent one which we finish "when we get around to it"

Reply:
As per the discussion yesterday during the SCREAM Microphysics Telecon, the units and variable description will be put into a separate LaTeX file. This file will include a table with entries for variable name, symbol, description and units.